### PR TITLE
Revamp UI of the Simulator [Task 1.1.1]✔️

### PR DESCRIPTION
**Pull request by abstrekt**
#### Description:
* ~~It's a work in progress.~~
* It's a part of this GSoC [project 4](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2720-Project-List#project-4---enhancing-simulator-user-interface).

### CODE REVIEW

File: public/css/0-helpers/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//*****************
+// Color Variables
+//*****************
+
+$charcoal: #454545;
+$silver: #BBBBBB;
+$gainsboro: #DDDDDD;
+$white: #FFFFFF;
+$grey : #7D7D7D;
+$very-light-grey: #CACACA;
```
[the remark this answers is not in the document]
Comment (abstrekt) [reply public/css/0-helpers/_variables.scss:10]: seriously

**Comment by abstrekt** [comment]
in the last commit I just added some styles, why is coverage being affted?

**Comment by nitin10s** [comment]
@abstrekt please resolve the conflicts, I've changed the base branch.

**Comment by abstrekt** [comment]
@nitin10s please check

**Pull request merged**